pootle/apps/virtualfolder/receivers.py

```diff
@@ -92,20 +92,22 @@ def vfolder_unit_priority_presave_handler(**kwargs):
                                 vfolders=original))
         for unit in removed_units.iterator():
             unit.vfolders.remove(original)
-            unit.set_priority()
             stores_affected.add(unit.store)
 
     for store in stores_affected:
         update_vfolder_tree(original, store)
+        store.set_priority()
```

Comment:
turns out this is necessary in this implementation - the signals will change anyway in the next related PR